src/components/pages/UserImagePage.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { FunctionComponent, useEffect, useState, useRef, useCallback } from 'react';
+
+import Dashboard from '../../scripts/clientUtils';
+import globalize from '../../scripts/globalize';
+import LibraryMenu from '../../scripts/libraryMenu';
+import { appHost } from '../apphost';
+import confirm from '../confirm/confirm';
+import ButtonElement from '../dashboard/users/ButtonElement';
+import UserPasswordForm from '../dashboard/users/UserPasswordForm';
+import loading from '../loading/loading';
+import toast from '../toast/toast';
+
+type IProps = {
+    userId: string;
+}
+
+const UserImagePage: FunctionComponent<IProps> = ({userId}: IProps) => {
```

Comment:
`UserProfilePage` seems like it would be a more accurate name for this page.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3073/commits/7af36ab52b4e6d4e587d868c36d9fd10b3c9d48e